_data/amit2020cs/amit2020cs.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Amit Kumar Singh 
+institution: Guru Govind Singh Indraprasth University
+profile_pic: Amit.jpg 
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `Amit.jpg` to `amit.jpg ` in the markdown file.
2. Rename `amit.jpg` to `Amit.jpg`.